src/components/form-elements/draft-js-mention-selector/DraftJSMentionSelectorCore.js

```diff
@@ -229,6 +234,27 @@ class DraftJSMentionSelector extends React.Component<Props, State> {
         return !!(activeMention && activeMention.mentionString && contacts.length);
     };
 
+    buildAccessibleAlert = (activeMention: Object) => {
```

Comment:
Should we consider a React approach instead of manually manipulating the DOM directly?

Reply:
Can you recommend me a react way to attack this?

Reply:
I guess it is the only way to do such a particular change

Reply:
Seems like what you're building on L247 could be an `AccessibilityAlert` component that takes some i18n message. 

To make it re-usable, maybe you could hide and show via CSS, or maybe play with the `key` React prop to make it unique each time it renders